framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -203,15 +213,38 @@ public Void visitDeclared(AnnotatedDeclaredType type, Void p) {
          * If any constant-value annotation has &gt; MAX_VALUES number of values provided, treats
          * the value as UnknownVal. Works together with ValueVisitor.visitAnnotation, which issues a
          * warning to the user in this case.
+         *
+         * <p>Additional rules applied for IntVal and IntRange annotations: if anno is IntVal and
```

Comment:
"anno" is a private variable, so the public documentation should not mention it but should reference atm.